static/js/addReviewClient.js

```diff
@@ -1,35 +1,89 @@
 const addReviewButton = document.querySelector(".add-review-button");
-let addReviewContent = document.querySelector(".add-review-content");
+const addReviewContent = document.querySelector(".add-review-content");
+const loggedOutEl = document.querySelector(".add-review-logged-out");
+const loggedInEl = document.querySelector(".add-review-logged-in");
 
 addReviewButton.addEventListener("click", () => {
   addReviewButton.classList.toggle("clicked");
   addReviewContent.classList.toggle("show");
 });
 
+let loggedIn = false;
+let sendReviewToken;
+
+document.querySelector("#login").addEventListener("click", async () => {
+
+  let username = document.querySelector("#username").value;
+  let password = document.querySelector("#password").value;
+  const bufferedInput = btoa(username + ":" + password);
+  try {
+    const response = await fetch(
+    new Request("/api/token", {
+      method: "POST",
+      headers: {
+        Accept: "application/json",
+        "Content-Type": "application/json",
+        "Authentication": 
+        "Basic " + bufferedInput,
+      },
+    }));
```

Comment:
you shouldn't have to use `Request` explicitly when using `fetch()`. These same lines of code could be written with a simple object, as:

```js
const response = await fetch('/api/token', {
  method: 'POST',
  headers: {
    //…
  }
});
```